tests/test_unit/test_cli/test_build_clean_snapshots/timeseries_equipment_sql.yaml

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
I realize now that this test is not working at all. You would expect the clean call to return identifier of the resources it would delete. 

I guess this test was never updated after we started to only delete existing resources.